src/components/MapsMirrorLayout.svelte

```diff
@@ -11,27 +11,23 @@
   const rowExceptions = [2, 6];
   let rowOrColumn;
 
-  $: sections = [maps];
-
   $: numberOfMaps = maps.length;
 
-  $: if (numberOfMaps) {
-    rowOrColumn = rowExceptions.includes(numberOfMaps) ? 'row' : 'column';
-  }
+  $: rowOrColumn = rowExceptions.includes(numberOfMaps) ? 'row' : 'column';
 
-  $: {
-    if (maps.length) {
-      const numOfSections = Math.round(Math.sqrt(numberOfMaps));
-      const mapsPerSection = Math.floor(numberOfMaps / numOfSections);
-      let mapArr = JSON.parse(JSON.stringify(maps));
-      let nextSections = [];
-      while (mapArr.length) {
-        nextSections.push(mapArr.slice(0, mapsPerSection));
-        mapArr = mapArr.slice(mapsPerSection);
-      }
-      sections = nextSections;
+  const getSections = maps => {
```

Comment:
I worry a little in instances like this that we should use a different variable name for the passed argument instead of `maps` which is also the top level var. Trying to weigh the pros and cons of a different name to make the scoped var more legible vs having fewer variable names floating around. Any thoughts?

Reply:
As long as the function is relatively short I'm fine with keeping the parameter name and variable name the same.